Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM lu4p/xgo-custom
+
+RUN apt-get update && apt-get upgrade -y && apt-get install -y tor build-essential libtool autopoint wget unzip
+RUN mkdir -p /go/src/github.com/cretz/
+RUN cd /go/src/github.com/cretz/ && git clone https://github.com/cretz/tor-static.git --recursive
+RUN cd /go/src/github.com/cretz/tor-static && go run build.go build-all
+RUN cd /go/src/github.com/cretz/tor-static && go run build.go package-libs
+RUN cd /go/src/github.com/cretz/tor-static && rm libs.zip && mv libs.tar.gz /libs_linux.tar.gz
+#get windows libs
+RUN cd /go/src/github.com/cretz/tor-static  && wget https://github.com/lu4p/tor-static/releases/download/2/tor-static-windows-amd64.zip
```

Comment:
I'm not sure I want these links to other repos in this repo. While I can see the value of a dockerfile to assist building (though I tried really hard to make it only a couple of lines), I fear requiring a not-popular image and a link to another repo's release section.

My hope w/ the build script is to make it easy to build w/out a container and my hope w/ the CI process (one day) is to do the building for people that may not have, e.g., Windows. In the meantime, I'm not convinced dockerizing it has that much value over someone just having their own Dockerfile since undoubtedly they'll be linking it with their Go project on build anyways.

Reply:
The Dockerfile is a helper for one of my own projects committed the file accidently to master after starting this pull request. A seperate pull request #28 without the Dockerfile is now online.